### DISCUSSION TITLE
Do we want to add firejail.list to the wiki? [yes, added]

### DISCUSSION THREAD

**Discussion by rusty-snake** [netblue30/firejail#4433] [Wiki]
I recently created [firejail.list – An incomplete list of projects that use or complement firejail](https://github.com/rusty-snake/firejail.list). Do we want to move this into the wiki?

Vote :+1: :-1:.

**Comment by kmk3** [comment]
> I recently created [firejail.list – An incomplete list of projects that use
> or complement firejail](https://github.com/rusty-snake/firejail.list). Do we
> want to move this into the wiki?
> 
> Vote +1 -1.

I think these links are really helpful for discovery; I also had started
collecting related projects on a list (though I only got a few).

I would just put the firejail-profile repos on their own list, as the projects
are of the same type/category and there may be many of such projects.  Maybe
one list for tools and another for custom configurations.

**Reply by rusty-snake** [reply]
https://github.com/rusty-snake/firejail.list/commit/bae2ad0723b91c5591f4efe907757fed2ce65280

**Reply by kmk3** [reply]
> [rusty-snake/firejail.list@bae2ad0](https://github.com/rusty-snake/firejail.list/commit/bae2ad0723b91c5591f4efe907757fed2ce65280)

Nicely done!

**Comment by rusty-snake** [comment]
Has anyone a better name for it? Third-party projects? Firejail universe?

**Reply by reinerh** [reply]
Both sound good to me.

**Comment by rusty-snake** [comment]
Done.
https://github.com/netblue30/firejail/wiki/Firejail-Universe
